### ISSUE TITLE
Reported issue with Outlook Express Classic

### ISSUE THREAD

**Issue by ShaneGowland** [MoscaDotTo/Winapp2#413]
Issue from CCEnhancer user that seems to be winapp2.ini-related:

> – CCEnhancer (CCE) wipes/removes the temporary folders & files created by Outlook Express Classic (OEC) and that creates a problem because the program wipes simply too much information.
> 
> CCE wipes everything in the
> 
> “C:\users\<—user—>\Appdata\Local\OEClassic\Prg\………” folder.
> 
> – It also removes the “T” subfolder (in the “Prg” subfolder, see above) and then OEC starts generating errors everytime the user wants to select an email.
> 
> –  In order to get OEC working again I had to manually re-create that folder “T” again.

**Comment by CSGalloway** [comment]
@ShaneGowland - I use OEC v OE Classic Pro 3.0 (10/9/2019). I have NO problems using the cleaning rule. what version do you use of OEC? Have you checked for updates - using Help -> Check For updates while inside the program?

**Comment by ROCKNROLLKID** [comment]
We can just add a ExcludeKey to the T folder if it is causing issues to some users.

**Comment by APMichael** [comment]
I have checked OE Classic (2.9) and can confirm the issue. But this only happens if you clean the folder while OE Classic is still running. (The cleaning of a running program can of course always lead to issues...)

I also found out that it's not the content that matters, only the folder itself. Therefore, the entry should use separate FileKeys per subfolder. The advantage is that all temporary files are still removed, but OE Classic no longer reports an error.

```
[OE Classic *]
LangSecRef=3022
Detect=HKCU\Software\OEClassic
Default=False
FileKey1=%LocalAppData%\OEClassic\Prg\Res|*.*|RECURSE
FileKey1=%LocalAppData%\OEClassic\Prg\T|*.*|RECURSE
```

My test installation shows only two subfolders. @CSGalloway, are there any more subfolders that need to be added?
